packages/flutter_tools/lib/src/globals.dart

```diff
@@ -118,6 +118,12 @@ ProcessUtils get processUtils => context.get<ProcessUtils>()!;
 const Platform _kLocalPlatform = LocalPlatform();
 Platform get platform => context.get<Platform>() ?? _kLocalPlatform;
 
+/// Contains messages produced by Flutter tools.
+//
+// This allows partial reimplementations of the flutter tool to override
+// certain messages.
+// TODO(andrewkolos): It is unclear if this is worth keeping. See
```

Comment:
we can delete the TODO, right?

Reply:
oh nvm, that was the class overall.

Reply:
The TODO is asking us to one day remove `UserMessages` altogether. Would this make more sense if I moved it to the class definition?

Reply:
Yeah, that would make more sense, lest I make this comment again 6 months from now :)